### PR TITLE
chore(operators): add cstor and csi 1.12.0 operator yamls

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

- cstor operators RC1,RC2 and GA release yamls
- csi operators RC1,RC2 and GA release yamls
- reverting the root mount changes for this release

### CODE REVIEW

File: 1.12.0/csi-operator-1.12.0-RC2.yaml [hunk 1]
```diff
@@ -871,27 +877,6 @@ roleRef:
   name: openebs-cstor-csi-registrar-role
   apiGroup: rbac.authorization.k8s.io
 
----
```
Comment (mittachaitu) [inline 1.12.0/csi-operator-1.12.0-RC2.yaml:874]: Don't we need to revert the changes made in [commit](https://github.com/openebs/cstor-csi/pull/101/files#diff-672f2ef4bc468f4edd0828699604b2c9).

Comment (prateekpandey14) [reply 1.12.0/csi-operator-1.12.0-RC2.yaml:874]: Those are reverted from 1.12.x branch

File: 1.12.0/csi-operator-1.12.0-RC1.yaml [hunk 2]
```diff
@@ -490,22 +490,6 @@ status:
...
-  # Supports persistent and ephemeral inline volumes.
-  volumeLifecycleModes:
-  - Persistent
-  - Ephemeral
-  # To determine at runtime which mode a volume uses, pod info and its
-  # "csi.storage.k8s.io/ephemeral" entry are needed.
-  podInfoOnMount: true
-  attachRequired: true
-
 ---
```
Comment (mittachaitu) [inline 1.12.0/csi-operator-1.12.0-RC1.yaml:499]: Q: In the starting of the file do we need to specify it supported only in so and so OS? As mentioned in other places?
```
# only with Ubuntu 16.04 and CentOS hosts, due to dependencies on
--
  | # linux distros. |   | # kernel components of iSCSI protocol.  For other ubuntu flavours and
  |   |   | # linux distros, this needs to be modified.
  |   |   | #
  |   |   | # Instructions to modify:
  |   |   | # Ubuntu-18.04 and above: https://github.com/openebs/cstor-csi/blob/master/deploy/iscsiadm-ubuntu-18.04-and-above-deps.yaml
  |   |   | # SUSE Linux Enterprise Server 12:  https://github.com/openebs/cstor-csi/blob/master/deploy/iscsiadm-suse-enterprise-server-12-deps.yaml
  |   |   | # SUSE Linux Enterprise Server 15:  https://github.com/openebs/cstor-csi/blob/master/deploy/iscsiadm-suse-enterprise-server-15-deps.yaml


```

File: 1.12.0/csi-operator-1.12.0-RC1.yaml [hunk 3]
```diff
@@ -490,22 +490,6 @@ status:
   conditions: []
   storedVersions: []
 
----
```
Comment (mittachaitu) [inline 1.12.0/csi-operator-1.12.0-RC1.yaml:493]: We need to remove `openebs-cstor-csi-iscsiadm` config map from this 1.12.0-RC1.yaml.

File: 1.12.0/csi-operator-1.12.0-ubuntu-18.04.yaml [hunk 4]
```diff
@@ -0,0 +1,996 @@
...
+              mountPath: /plugin
+            - name: registration-dir
+              mountPath: /registration
+        - name: cstor-csi-plugin
+          securityContext:
+            privileged: true
+            capabilities:
+              add: ["CAP_MKNOD", "CAP_SYS_ADMIN", "SYS_ADMIN"]
+            allowPrivilegeEscalation: true
+          image: openebs/cstor-csi-driver:1.12.0-RC2
```
Comment (mittachaitu) [inline 1.12.0/csi-operator-1.12.0-ubuntu-18.04.yaml:929]: Change image to `openebs/cstor-csi-driver:1.12.0`

File: 1.12.0/csi-operator-1.12.0-RC1.yaml (hunk 3 above)
Comment (prateekpandey14) [reply 1.12.0/csi-operator-1.12.0-RC1.yaml:493]: Please refresh the page changes are already there ?

File: 1.12.0/csi-operator-1.12.0-RC1.yaml (hunk 2 above)
Comment (prateekpandey14) [reply 1.12.0/csi-operator-1.12.0-RC1.yaml:499]: That's fine, i think its good to have this info there, in a single file.

File: 1.12.0/csi-operator-1.12.0-ubuntu-18.04.yaml (hunk 4 above)
Comment (prateekpandey14) [reply 1.12.0/csi-operator-1.12.0-ubuntu-18.04.yaml]: done

File: cstor-operator.yaml [hunk 5]
```diff
@@ -336,7 +467,7 @@ metadata:
   labels:
     name: cspc-operator
     openebs.io/component-name: cspc-operator
-    openebs.io/version: 1.11.0
+    openebs.io/version: 1.12.0
```
Comment (mittachaitu) [inline cstor-operator.yaml:470]: Why do we need to change the version for this file?

Comment (prateekpandey14) [reply cstor-operator.yaml:470]: These are the files which are maintained outside of  version dir,  similar to openebs-operator or other yaml, as a latest version release yaml
